src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -133,7 +133,13 @@ public class AdminController : Controller
                 // Allows non creatable types to be created by another admin page.
                 if (contentTypeDefinition.GetSettings<ContentTypeSettings>().Creatable || options.CanCreateSelectedContentType)
                 {
-                    creatableList.Add(new SelectListItem(new LocalizedString(contentTypeDefinition.DisplayName, contentTypeDefinition.DisplayName).Value, contentTypeDefinition.Name));
+                    var contentItem = await _contentManager.NewAsync(contentTypeDefinition.Name);
+                    contentItem.Owner = context.User.FindFirstValue(ClaimTypes.NameIdentifier);
+
+                    if (await _authorizationService.AuthorizeAsync(context.User, CommonPermissions.EditContent, contentItem))
```

Comment:
Should this be EditOwnContent ?

Reply:
No it is implied by EditContent

Reply:
But what about when someone has EditOwn but not Edit ? He would not see the new button no ?

Reply:
They're morphed @jptissot 

You (in your code, or code here) should never request or think about EditOwn, always request Edit

Reply:
I see, thanks for the clarification!